README.md

```diff
@@ -157,32 +158,39 @@ source venv/bin/activate
 pip install -r requirements/production.txt
 ```
 
-Export the environment for the `manage.py` commands:
+Export the environment for the `manage.py` commands and go to the 'accesspoc' directory:
 
 ```
 export $(cat ~/accesspoc-env)
+cd accesspoc
 ```
 
 Initialize the database:
 
 ```
-accesspoc/manage.py migrate
+./manage.py migrate
 ```
 
 Create search indexes:
 
 ```
-accesspoc/manage.py index_data
+./manage.py index_data
 ```
 
 Add a superuser:
 
 ```
-accesspoc/manage.py createsuperuser
+./manage.py createsuperuser
 ```
 
 Follow the instructions to create a user with full admin rights.
 
+Compile translation files:
+
+```
+django-admin compilemessages
```

Comment:
There's also `./manage.py compilemessages`, any reason to introduce `django-admin` to the user?

Reply:
Oh, I didn't knew that, the docs only talk about `django-admin` in that section. Thanks @sevein!!